docs/release-notes/8.0.32-24.md

```diff
@@ -0,0 +1,53 @@
+# Percona Server for MySQL 8.0.32-24 (2023)
+
+<style>
+    table {
+        width=50%';
+        font-family: Chivo, Colfax, Franziska, Helvetica, Arial, sans-serif;
+    }
+    table td {
+        border: 0px;
+        padding: 8px;
+    }
+    tr:nth-child(even){
+        background-color:#f5f5f5
+    }
+    tr:hover {
+        background-color: #dddd;
+    }
+</style>
+
+| Release date | 2023 |
+|---|---|
+| Install instructions | [Install Percona Server for MySQL](https://docs.percona.com/percona-server/8.0/installation.html) |
+| Download this version | [Percona Server for MySQL](https://www.percona.com/downloads/Percona-Server-LATEST/)
+
+[Percona Server for MySQL 8.0.32-24](https://www.percona.com/software/mysql-database/percona-server) includes all the features and bug fixes available in the
+[MySQL 8.0.32 Community Edition](https://dev.mysql.com/doc/relnotes/mysql/8.0/en/news-8-0-32.html)
+in addition to enterprise-grade features developed by Percona.
+
+Percona Server for MySQL is a free, fully compatible, enhanced, and open
+source drop-in replacement for any MySQL database. It provides superior
+performance, scalability, and instrumentation.
+
+Percona Server for MySQL is trusted by thousands of enterprises to provide
+better performance and concurrency for their most demanding workloads. It
+delivers more value to MySQL server users with optimized performance,
+greater performance scalability and availability, enhanced backups, and
+increased visibility. 
+
+For [training](https://www.percona.com/training), contact [Percona Training - Start learning now](https://learn.percona.com/contact-me)
+
+## Release highlights
+
+Percona decided to revert the following MySQL bug fix:
+
+    The data and the GTIDs backed up by mysqldump were inconsistent when the options --single-transaction and --set-gtid-purged=ON were both used. It was because in between the transaction started by mysqldump and the fetching of GTID_EXECUTED, GTIDs on the server could have increased already. With this fixed, a FLUSH TABLES WITH READ LOCK is performed before the fetching of GTID_EXECUTED to ensure its value is consistent with the snapshot taken by mysqldump.
+
+    The current Percona decision also provides a solution for [MySQL bug #33630199](https://bugs.mysql.com/bug.php?id=109701).
+
```

Comment:
Add this info to "Improvements and bug fixes introduced by Oracle for MySQL 8.0.32 and included in Percona Server for MySQL are the following":

```
* A replica can add a Generated Invisible Primary Keys(GIPK) to any InnoDB table. To achieve this behavior, the GENERATE value is added as a possible value for the CHANGE REPLICATION SOURCE TO statement’s REQUIRE_TABLE_PRIMARY_KEY_CHECK option.

REQUIRE_TABLE_PRIMARY_KEY_CHECK = GENERATE option can be used on a per-channel basis.

Setting sql_generate_invisible_primary_key on the source is ignored by a replica because this variable is not replicated. This behavior is inherited from the previous releases.
```

Reply:
Done.